vendor/engines/sanger_sequencing/app/views/sanger_sequencing/purchase_notifier/_samples_table.html.haml

```diff
@@ -6,10 +7,12 @@
       %tr
         %th= SangerSequencing::Sample.human_attribute_name(:customer_sample_id)
         %th= SangerSequencing::Sample.human_attribute_name(:id)
-        - if submission.has_results_files?
-          %th= SangerSequencing::Sample.human_attribute_name(:results_files)
+        - if needs_primer
+          %th= SangerSequencing::Sample.human_attribute_name(:primer)
```

Comment:
Is it right to remove the Result Files column header?

Reply:
I guess so, there's no data cell in the table that renders the result. This is show in the order receipt